CHANGELOG.md

```diff
@@ -19,6 +19,20 @@ We introduced this change in:
 - [#5499: Rename GOVUKFrontendComponentConfigurable, export ConfigurableComponent](https://github.com/alphagov/govuk-frontend/pull/5499)
 - [#5456: Refactor Accordion to extend from a GOVUKFrontendConfigurableComponent](https://github.com/alphagov/govuk-frontend/issues/5456)
 
+### Deprecated features
+
+#### Importing Sass using `govuk/all`
+
+You'll see a warning when compiling your Sass if you import all of GOV.UK Frontend's styling via `govuk/all`. Importing using the `all` file is deprecated, and we’ll remove it in the next major release.
+
+In your import statements, use a trailing `/index` rather than `/all` to load GOV.UK Frontend's files:
+
+- `@import "govuk/index";` instead of `@import "govuk/all";`;
+
+You do not need `/index` at the end of each import path if you’re using Dart Sass, LibSass 3.6.0 or higher, or Ruby Sass 3.6.0 or higher.
```

Comment:
**suggestion** Could we use this as an opportunity to remind that people should move to use Dart Sass rather than stay on LibSass or Ruby Sass or would that be too distracting?

```suggestion
You do not need `/index` at the end of each import path if you’re using Dart Sass (as well as LibSass 3.6.0 or higher, or Ruby Sass 3.6.0 or higher, though we recommend you migrate to Dart Sass if you're using LibSass or Ruby Sass as they are both deprecated).
```

Or maybe by just silencing that you have benefits in the deprecated Sass compilers?

```suggestion
You do not need `/index` at the end of each import path if you’re using Dart Sass.
```

Reply:
I think I prefer your second suggestion. Suggestion via erasure. It delivers the message (that messag being 'use Dart Sass') more subtly but also more efficiently. I'll wait and see what the content folks say.